src/login/msal/PublicClientCredential.ts

```diff
@@ -38,4 +40,15 @@ export class PublicClientCredential implements TokenCredential {
 
 		return null;
 	}
+
+	public async signRequest(webResource: WebResource): Promise<WebResource | undefined> {
+		const tokenResponse: AccessToken | null = await this.getToken(defaultMsalScopes);
+		if (tokenResponse) {
+			webResource.headers.set(
+				MSRestConstants.HeaderConstants.AUTHORIZATION,
+				`${MSRestConstants.HeaderConstants.AUTHORIZATION_SCHEME} ${tokenResponse.token}`
```

Comment:
Is the auth scheme part of the token response or will it always be fixed?

Reply:
The scheme is fixed - it'll always be "Bearer". Would it be better to hard code that instead of using the MS Rest constant?

Reply:
I think it's fine to use the constant. I've seen in some places that response to the token request includes the type in addition to the token--so, it could be e.g. `Authorization: Bearer abcd123` or `Authorization: JWT 123abcd`, with the token response including the type `Bearer` or `JWT`. But if the token response has no such field, I'd say what you have is the way to go.